its/ruling/src/test/resources/eclipse-jetty/java-S6212.json

```diff
@@ -71,6 +71,10 @@
 590,
 592,
 599,
+842,
+844,
```

Comment:
Same FP here, `code` is used:
```
            HttpStatus.Code code = HttpStatus.getCode(i);
            if (code == null)
                continue;
            String reason = code.getMessage();
```

Reply:
Similar issue here

Reply:
I confused S6212 result with S1481. It's not a S1481 false-positive.